### DISCUSSION TITLE
Can cetz be loaded directly from subdirectory of the project?

### DISCUSSION THREAD

**Discussion by linuxrider** [cetz-package/cetz#413] [Q&A] [answered]
I cloned the repository in a subfolder of the project and tried to load cetz from the subfolder. 
It got stuck somewhere with the `process.typ` being not found.
Is it possible to load cetz from a subfolder?

Background: use non-released version of cetz in the webapp

**Comment by johannes-wolf** [comment, marked as answer]
Yes it can (with `0.2.0`), but you have to change all absolute import paths to relative ones. There is a script for that in the `scripts` directory.

If you have `just` installed, you can (from your cetz clone _outsides_ your project) call: `just package <project-path-to-install-cetz-to> --relative-paths`.

Or run the `./scripts/relpaths <your-cetz-root-dir>` Python script.

**Reply by linuxrider** [reply]
Thank you very much! It `just` worked.
